server/app/models/stack_revision.rb

```diff
@@ -12,6 +12,7 @@ class StackRevision
   field :expose, type: String
   field :services, type: Array
   field :volumes, type: Array
+  field :metadata, type: Hash
```

Comment:
Maybe add `default: {}` .

Reply:
Thought about it but variables didn't have one either.

Reply:
I think this should be fixed. Otherwise api will send `null` or `{ .. }` and it's quite ugly to handle on client side.